api_tests/nodes/views/test_node_draft_registration_detail.py

```diff
@@ -14,7 +14,8 @@
 )
 from rest_framework import exceptions
 from test_node_draft_registration_list import DraftRegistrationTestCase
-from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+
+SCHEMA_VERSION = 2
```

Comment:
"LATEST_SCHEMA_VERSION" isn't a special thing anymore.  As the versions of the schemas below change, this variable might need to change. Hardcoding in the test file instead.